tools/pipeline-witness/Azure.Sdk.Tools.PipelineWitness/BlobUploadProcessor.cs

```diff
@@ -310,52 +310,48 @@ private async Task UploadLogLinesBlobAsync(string account, Build build, BuildLog
                     return;
                 }
 
-                using var tempFile = new DisposableTempFile();
+                var messagesWriter = new StringBuilder();
+                var logLines = await this.logProvider.GetLogLinesAsync(build, log.Id);
+                var lastTimeStamp = log.CreatedOn;
 
-                await using (var messagesWriter = new StreamWriter(File.OpenWrite(tempFile.Path)))
+                for (var lineNumber = 1; lineNumber <= logLines.Count; lineNumber++)
                 {
-                    var logLines = await this.logProvider.GetLogLinesAsync(build, log.Id);
-                    var lastTimeStamp = log.CreatedOn;
+                    var line = logLines[lineNumber - 1];
```

Comment:
Nit: Why start at 1?

Reply:
Azure DevOps numbers the lines starting with 1:
![image](https://user-images.githubusercontent.com/1291634/155809750-2617948b-349d-4849-b9b8-c475795c840e.png)